presto-main/src/main/java/com/facebook/presto/operator/SimplePagesHashStrategy.java

```diff
@@ -61,6 +74,16 @@ public SimplePagesHashStrategy(
             this.precomputedHashChannel = null;
         }
         this.sortChannel = requireNonNull(sortChannel, "sortChannel is null");
+        this.functionRegistry = requireNonNull(functionRegistry, "functionRegistry is null");
```

Comment:
If I understand correctly, `functionRegistry` is just used in the constructor so we don't need to store it as a field member.